noxfile.py

```diff
@@ -92,18 +92,24 @@ def _create_requirements_pr_message(uv_output: str) -> None:
     ]
 
     for package_update in uv_output.splitlines():
-        if package_update.startswith("Resolved"):
+        if not package_update.startswith("Updated"):
+            session.debug(f"Line not added to table: {package_update}")
```

Comment:
There might be some standard output from `uv lock --upgrade` that this function is currently not handling, so it'll be helpful to print it out as a debug statement.